tests/ga/test_extension.py

```diff
@@ -294,6 +300,7 @@ def tearDownClass(cls):
             CGroupConfigurator.get_instance().disable()
 
 
+@patch('time.sleep', side_effect=lambda _: mock_sleep(0.001))
```

Comment:
Added a mock sleep to this class to reduce the time it took for running test cases

Reply:
This is awesome! Our unit test suite is now executing in ~1 minute instead of 7!

Reply:
haha thanks :)

Reply:
1 min!!!! Dude, I need to try this myself -- if True, I'll buy you a beer.

Reply:
Just check out his Travis run times: https://travis-ci.org/Azure/WALinuxAgent/builds/583298862?utm_source=github_status&utm_medium=notification
Laveesh deserves all the beers 😄

Reply:
haha always down for a beer 🍻